response_operations_ui/views/messages.py

```diff
@@ -176,6 +234,16 @@ def _refine(message):
     }
 
 
+def _get_survey_id(selected_survey):
+    survey_messages = survey_controllers.get_survey(selected_survey)
+    return survey_messages['survey']['id']
+
+
+def _get_FDI_survey_id():
+    fdi_surveys = ["AOFDI", "AIFDI", "QIFDI", "QOFDI"]
```

Comment:
I think these values could also go in an enum or at least with the other survey data